docs/geedocs/5.2.3/answer/4485225.html

```diff
@@ -385,12 +385,12 @@ <h4>Usage</h4>
   <code><strong>gerewritedbroot</strong></code> <code>--source=<em>server_name</em> --icon_directory=<em>directory</em><br/>
     --dbroot_file=<em>filename</em>  --kml_map_file=<em>filename</em><br/>
     [--search_service=<em>search_service_url</em>] [--kml_server=<em>server_name]</em><br/>
-    [--kml_port=<em>num]</em> [--kml_url_path=<em>prefix]</em> [--use_ssl_for_kml=<em>bool</em>]</code>
+    [--kml_port=<em>num]</em> [--kml_url_path=<em>prefix]</em> [--use_ssl_for_kml=<em>bool</em>] [--preserve_kml_filenames=<em>bool</em>]</code>
   <h4>Description</h4>
   <ul>
-    <li>Reads the dbRoot and rewrites the search tabs so that they point at the given search server and port.</li>
-    <li>Creates a directory of all of the icons referred to by the dbRoot.</li>
-    <li>Creates a file listing all KML files that are referenced in the dbRoot and gives their new name on the KML server.</li>
+    <li>Reads dbRoot and rewrites the search tabs so that they point to the given search server, port and path.</li>
+    <li>Optionally creates a directory of all of the icons referred to by the dbroot. </li>
+    <li>Optionally updates the host, path and/or filename to the network linked KML content in dbRoot. </li>
```

Comment:
* I think you should keep an item that says a file listing the correspondence between KML URLs from the old dbRoot, and the file names from the new dbRoot can be created.

* I don't think `gerewritedbroot` does any processing of KML network links.  This item should say that it can rewrite the hostname, and path of the KML URLs in KML wrapper layers.